package-lock.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
do we want to add this new file?

Reply:
I thought so when I added it but it ought to be pulled and I'll do so.